app/(dashboard)/mailboxes/[mailbox_slug]/settings/tools/toolSetting.tsx

```
@@ -15,7 +16,7 @@ type ToolSettingProps = {
 
 const ToolSetting = ({ mailboxSlug }: ToolSettingProps) => {
   const [showApiForm, setShowApiForm] = useState(false);
-  const { data: apis = [], isLoading: apisLoading, error } = api.mailbox.tools.list.useQuery({ mailboxSlug });
+  const { data: apis = [], isLoading: apisLoading, isFetching: apisFetching, error } = api.mailbox.tools.list.useQuery({ mailboxSlug });
```

ESLint

[error] Replace `·data:·apis·=·[],·isLoading:·apisLoading,·isFetching:·apisFetching,·error` with `⏎····data:·apis·=·[],⏎····isLoading:·apisLoading,⏎····isFetching:·apisFetching,⏎····error,⏎·`

(prettier/prettier)